test/widgets/conftest.py

```diff
@@ -74,3 +77,44 @@ class MinimalConf(libqtile.confreader.Config):
         screens = []
 
     return MinimalConf
+
+
+@pytest.fixture(scope='function')
+def dbus(monkeypatch):
+    # for Github CI, dbus-launch is provided by "dbus-x11" package
```

Comment:
Could you change this to say Ubuntu? That way if packagers are trying to figure out why their test is failing they'll find it easier to figure out how they can fix it rather than also figuring out how we configure the CI